admin/templates/users/user.html

```diff
@@ -0,0 +1,146 @@
+            </thead>
+            <tbody>
+                <tr>
+                    <td>Name</td>
+                    <td>{{ user.name }}</td>
+                </tr>
+                <tr>
+                    <td>OSF account</td>
+                    <td>
+                        <a href="">{{ user.id }}</a>
+                    </td>
+                </tr>
+                <tr>
+                    <td>Email</td>
+                    <td>
+                        {% for email in user.emails %}
+                            <li>
+                                {% if forloop.first %}
+                                    <strong>Primary: </strong>
+                                    {{ email }}
+                                    <a href=""
+                                       class="btn btn-default disabled">Email</a>
+                                {% else %}
+                                    {{ email }}
+                                {% endif %}
+                            </li>
+                        {% endfor %}
+                    </td>
+                </tr>
+                <tr>
+                    <td>Last login</td>
+                    <td>{{ user.last_login }}</td>
+                </tr>
+                <tr>
+                    <td>Account disabled</td>
+                    <td>False</td>
+                </tr>
+                <tr>
+                    <td>Two factor</td>
+                    <td>
+                        {{ user.two_factor }}
+                        {% if user.two_factor %}
+                            <button data-toggle="modal"
+                                    data-target="#twoFactorModal"
+                                    class="btn btn-default">
+                                Deactivate
+                            </button>
+                            <div class="modal" id="twoFactorModal"
+                                 style="display: none;">
+                                <div class="modal-dialog">
+                                <div class="modal-content">
+                                    <div class="modal-header">
+                                        <h3>
+                                            This will remove two factor auth.
+                                        </h3>
+                                    </div>
+                                    <div class="modal-body">
+                                        Removing this will let the user log in
+                                        with only one form of authentication.
+                                        The only way to reverse this action is
+                                        for the user to re-enable the addon
+                                        after logging in.
+                                    </div>
+                                    <div class="modal-footer">
+                                        <a href="{% url 'users:remove2factor' user.id %}"
```

Comment:
So far as I can tell, Django doesn't let you specify a request method via the templating language, so the best way to implement that may be via a [redirect](http://django.readthedocs.org/en/1.9.x/topics/http/urls.html#examples) that does some processing on it before altering the method and sending it along to `remove2factor`. That's for ADM-79, though